app/controllers/api/services_controller.rb

```diff
@@ -37,9 +37,9 @@ def settings
   def create
     @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
     @service.attributes = service_params
```

Comment:
Those should be set in the CreateService object

Reply:
@hallelujah @duduribeiro Ignore this for now plz because it is fixed in my PR and u would make me have more conflicts unnecessarily if u change it also here 😝